src/main/java/network/radicle/tools/github/commands/Command.java

```diff
@@ -0,0 +1,81 @@
+package network.radicle.tools.github.commands;
+
+import jakarta.enterprise.context.Dependent;
+import jakarta.inject.Inject;
+import network.radicle.tools.github.Config;
+import network.radicle.tools.github.services.MigrationService;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import picocli.CommandLine;
+
+@Dependent
+@CommandLine.Command
+public class Command implements Runnable {
+    @CommandLine.Option(
+            names = {"-gv", "--github-api-version"},
+            defaultValue = "github.api.version",
+            description = "The version of the GitHub REST API (e.g. 2022-11-28)")
+    String gVersion;
+
+    @CommandLine.Option(
+            names = {"-gu", "--github-api-url"},
+            defaultValue = "github.api.url",
+            description = "The base url of the GitHub REST API")
+    String gUrl;
+
+    @CommandLine.Option(
+            names = {"-gr", "--github-repo"},
+            required = true,
+            defaultValue = "${GITHUB_REPO}",
+            description = "The target GitHub repo")
+    String gRepo;
+
+    @CommandLine.Option(
+            names = {"-go", "--github-repo-owner"},
+            required = true,
+            defaultValue = "${GITHUB_OWNER}",
+            description = "The owner of the target GitHub repo")
+    String gOwner;
+
+    @CommandLine.Option(
+            names = {"-gt", "--github-token"},
+            required = true,
+            interactive = true,
+            defaultValue = "${GITHUB_TOKEN}",
+            description = "The GitHub authentication token")
+    String gToken;
+
+    @ConfigProperty(name = "github.api.page-size")
```

Comment:
Either convert this to a constant, or make it cli configurable.
I think that having different configuration methods makes it hard to reason about.